src/main/java/edu/cornell/kfs/concur/ConcurKeyConstants.java

```diff
@@ -3,4 +3,14 @@
 public class ConcurKeyConstants {
     public static final String CONCUR_ACCOUNT_INFO_IS_REQUIRED = "error.concur.account.info.required";
     public static final String INCORRECT_CONCUR_STATUS_CODE = "error.concur.status.invalid";
+
+    public static final String CONCUR_REQUEST_EXTRACT_NOT_CASH_ADVANCE_DATA_LINE = "validation.error.concur.request.extract.not.cash.advance";
+    public static final String CONCUR_REQUEST_EXTRACT_CASH_ADVANCE_USED_IN_EXPENSE_REPORT = "validation.error.concur.request.extract.cash.advance.in.expense.report";
+    public static final String CONCUR_REQUEST_EXTRACT_REQUEST_ID_INVALID = "validation.error.concur.request.extract.invalid.request.id";
+    public static final String CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NULL_OR_BLANK = "validation.error.concur.request.extract.invalid.employee.id";
+    public static final String CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NOT_FOUND_IN_KFS = "validation.error.concur.request.extract.employee.id.not.found";
+    public static final String CONCUR_REQUEST_EXTRACT_PAYEE_ID_TYPE_INVALID = "Payee ID type was not specified as EMPLOYEE or NON-EMPOYEE.";
```

Comment:
Daniela: 
I realize the unit tests are failing but I wanted to get this code out there for you to look at.